PlayCover/Views/Sidebar Views/AppFolderView.swift

```diff
@@ -0,0 +1,272 @@
+//
+//  AppLibraryView.swift
+//  PlayCover
+//
+
+import SwiftUI
+import DataCache
+
+struct AppFolderView: View {
```

Comment:
Is it necessary to create a new View for this, or is it possible to re-use existing code? I haven't tested this yet so let me know.

Reply:
This view has some difference from the "normal" one like the "edit" button. Was also thinking to change the "add app" so app will automatically added to list of apps in that folder. But i can do all with a flag if is better